packages/mobile/App/models/PatientProgramRegistrationCondition.ts

```diff
@@ -61,14 +61,14 @@ export class PatientProgramRegistrationCondition
   @RelationId(({ deletionClinician }) => deletionClinician)
   deletionClinicianId?: ID;
 
-  static async findForRegistryAndPatient(programRegistryId: string, patientId: string) {
+  static async findForRegistryAndPatient(patientProgramRegistrationId: string) {
```

Comment:
```suggestion
  static async findForRegistration(patientProgramRegistrationId: string) {
```
Might be nice to rename here given args change